x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/GetDataFrameAnalyticsStatsAction.java

```diff
@@ -14,22 +14,22 @@
 import org.elasticsearch.action.support.tasks.BaseTasksRequest;
 import org.elasticsearch.action.support.tasks.BaseTasksResponse;
 import org.elasticsearch.cluster.node.DiscoveryNode;
-import org.elasticsearch.core.Nullable;
```

Comment:
As there were no actual changes to this file, should we revert it so it is not changed by this PR?

Reply:
I probably hit the organise imports shortcut by accident, the imports are now in the right order now anyway so that's better.